Action.c

```diff
@@ -410,6 +410,25 @@ static Htop_Reaction actionStrace(State* st) {
    return HTOP_REFRESH | HTOP_REDRAW_BAR;
 }
 
+static Htop_Reaction actionDebugger(State* st) {
+   Settings* settings = st->settings;
+   Process* p = (Process*) Panel_getSelected(st->panel);
+   if (!p) return HTOP_OK;
+
+   char buf[32];
```

Comment:
Probably a bigger buffer should be used, if a user configures a long debugger path, like
`/home/myusername/development/projects/debugger/bin/my-debugger`

Reply:
Yes.